app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>MovieApp</title>
```

Comment:
Create a story to integrate i18n into your project

Learning materials: 
- https://guides.rubyonrails.org/i18n.html
- https://github.com/svenfuchs/rails-i18n